src/views/SendConfirm.js

```diff
@@ -90,13 +91,7 @@ const SendConfirm = () => {
   );
 
   const isAmountTooHigh = amount
-    ? web3.utils
-        .toBN(maxAmount)
-        .lte(
-          web3.utils.toBN(
-            amount ? web3.utils.toWei(amount.toString(), 'ether') : '0',
-          ),
-        )
+    ? Number(formatCirclesValue(maxAmount)) < Number(amount)
```

Comment:
why don't you preserve the comparison in BN?, it should be equivalent as the timestamp is now() for both quatities

Reply:
I think it's safer too compare in Big numbers and not in numbers, so that you don't have to care about decimal inconsistencies. at the end of the day the number in the blockchain is what matter, and the timeCircles conversion functions are consistent (same query, same result)